.azure-pipelines/client.yml

```diff
@@ -75,6 +75,26 @@ jobs:
 
       - script: |
           node common/scripts/install-run-rush.js install
+        displayName: 'Install dependencies'
+
+      - script: |
+          node common/scripts/install-run-rush.js lint
+        displayName: 'Run lint'
+
+      - task: CopyFiles@2
+        inputs:
+          contents: 'sdk/**/**/*lintReport.html'
+          targetFolder: $(Build.ArtifactStagingDirectory)
+          flattenFolders: true
+        displayName: 'Copy lint reports'
+
+      - task: PublishBuildArtifacts@1
+        condition: succeededOrFailed()
+        displayName: 'Publish lint reports'
+        inputs:
+          artifactName: lintReports
```

Comment:
For consistency with other pipelines lets just call this artifact "reports".

Reply:
done